trend_micro_vision_one_endpoint_security/assets/logs/trend-micro-vision-one-endpoint-security.yaml

```diff
@@ -0,0 +1,2633 @@
+            0,Unknown
+            1,Information
+            2,Warning
+            3,Error
+            4,Critical
+          type: lookup-processor
+        - name: Lookup for `cn2` to `risk_level` field
+          enabled: true
+          source: cn2
+          target: risk_level
+          lookupTable: |-
+            100,High
+            300,Medium high
+            500,Medium
+            700,Medium low
+            900,Low
+          type: lookup-processor
+        - name: Lookup for `action` to `action` field
+          enabled: true
+          source: action
+          target: action
+          lookupTable: |-
+            0,Unknown
+            1,Pass
+            2,Block
+            3,Monitor
+            4,Delete
+            5,Quarantine
+            6,Warn
+            7,Warn and continue
+            8,Override
+          type: lookup-processor
+        - name: Lookup for `filter_blocking_type` to `filter_blocking_type` field
+          enabled: true
+          source: filter_blocking_type
+          target: filter_blocking_type
+          lookupTable: |-
+            0,Unknown
+            1,File name
+            2,Webmail site
+            3,Web server
+            4,URL pattern
+            5,Java/VB script
+            6,True file type
+            7,User-defined
+            8,Server-defined
+            9,Web policy
+            11,Phishing
+            12,Phishing/Spyware/Grayware
+            13,Phishing/Virus/Malware accomplice
+            14,Phishing/Forged signature
+            15,Phishing/Disease vector
+            16,Phishing/Malicious applet
+            17,Phishing reputation
+            20,IP translation policy
+            21,Java scanning policy
+            22,Malicious mobile code policy
+            31,Pharming
+            32,URL blocking
+            33,URL filtering
+            34,Client IP blocking
+            35,Destination port blocking
+            36,Web reputation
+            41,Unsupported file type
+            42,Exceeds total file count limit
+            43,Exceeds file size limit
+            44,Exceeds decompression layer limit
+            45,Exceeds decompression time frame
+            46,Exceeds compression ratio limit
+            47,Password protected file
+            48,Restricted spyware/grayware type
+            60,String pattern
+            70,HTTP inspection
+            -1,Virus/Malware
+            -2,Spyware/Grayware
+            -3,Network virus
+            -4,IntelliTrap
+            -5,Suspicious virus/malware
+            -6,Suspicious spyware/grayware
+            -7,Fraud
+            -8,Suspicious behavior
+          type: lookup-processor
+    - type: grok-parser
+      name: Convert `senders` field into an array
+      enabled: true
+      source: senders
+      samples:
+        - user1@example.com
+        - user1@example.com;user2@example.com
+      grok:
+        supportRules: ""
+        matchRules: senders_rule %{data:senders:array("", ";")}
```

Comment:
Hi @mathieu-charbonnel, As per [this](https://docs.datadoghq.com/service_management/events/pipelines_and_processors/grok_parser/?tab=filters#list-to-array) datadog documentation using a data is the only option to extract `;` separated values into array field. Let us know if you think otherwise.

Reply:
I am guessing that  %{notSpace:senders:array("", ";")} would work if there is no space in the email list right ?

Reply:
Yes, it's working. I updated the rule with `notSpace`.

Reply:
Cool, I move this to testing in staging